PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -22,7 +22,7 @@ public static void Main(string[] args)
             Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
         }
 
-        public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
+        public static string GetPercentCorrect(float numberCorrectAnswers, float numberOfQuestions)
```

Comment:
Interesting solution to just change the parameter type. Did not think of this one. One change I would have made, though, would be to add a Math.Round in here in case after the division you still had a number that wasn't whole.
```suggestion
return Math.Round((float)numberOfCorrectAnswers / numberOfQuestions * 100) + “%”
```